pages/common/phan.md

```diff
@@ -0,0 +1,32 @@
+# phan
+
+> A static analysis tool for PHP.
+> More information: <https://github.com/phan/phan>.
+
+- Generate a `.phan/config.php` in the current directory:
+
+`phan --init`
+
+- Generate a Phan configuration file using a specific level (1 being strictest to 5 being the least strict):
+
+`phan --init --init-level {{level}}`
+
+- Analyse the current directory:
+
+`phan`
+
+- Analyse one or more directories:
```

Comment:
space separated list of directories?

Reply:
Looks like you have to chain the `--directory` flag. So:
`phan --directory {{path/to/directory1}} --directory {{path/to/directory2}}`